OneCommunityGlobal/HighestGoodNetworkApp
# Masasa - applied all name and description updates

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
This PR applies the update about name and description updates on the permission page such that they match the names in this [spreadsheet](https://docs.google.com/spreadsheets/d/1TfiJY9OLDZuyP2UgF0C1wI9tOSLKbS9MiJFH6n1Dsao/edit#gid=0)
<img width="264" alt="Screenshot 2023-08-19 at 4 10 11 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/ceeff0f7-84f2-4ec1-9ed7-e1b013a7ed46">


## Related PRS (if any):
No related PRs
…

## Main changes explained:
changed the naming in for permission labels in UserRoleTab.jsx
Changed the corresponding descriptions in RolePermission.jsx

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user
4. go to OtherLinks→ Permissions Management→ choose user role→…
5. Make sure the names and descriptions have been updated

## Screenshots or videos of changes:
<img width="669" alt="Screenshot 2023-08-19 at 4 19 55 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/fac36782-4298-43ed-aa20-30f70eb8ff4c">
<img width="712" alt="Screenshot 2023-08-19 at 4 20 05 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73863365/1a923c10-d2f7-42b5-a06d-180d472e38db">
